CorsixTH/Lua/dialogs/fullscreen/fax.lua

```diff
@@ -166,6 +166,7 @@ function UIFax:choice(choice_number)
         owner:goHome("over_priced", owner.disease.id)
       end
     elseif choice == "research" then
+      owner:unregisterCallbacks()
       owner:setMood("idea", "activate")
       owner:setNextAction(SeekRoomAction("research"))
     end
```

Comment:
This change I believe fixes the last issue in #133. Believe in this issue the callback may still be registered so when the edited room exists again the callback was executed requeuing the seek to the room. Yet to attempt to verify